daliuge-engine/dlg/lifecycle/dlm.py

```diff
@@ -482,8 +483,8 @@ def handleCompletedDrop(self, uid):
             return
 
         drop = self._drops[uid]
-        if drop.precious and self.isReplicable(drop):
-            logger.debug("Replicating %r because it's precious", drop)
+        if drop.persist and self.isReplicable(drop):
```

Comment:
It wasn't clear to me if this ticket covers changing the implementation of `isReplicable()` to be allowed by default? (probably should be in a separate PR anyway)